src/table.ts

```diff
@@ -810,33 +810,34 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
 
       requestStream!.on('request', () => numRequestsMade++);
 
-      rowStream = pumpify.obj([
-        requestStream,
-        chunkTransformer,
-        through.obj((rowData, enc, next) => {
-          if (
-            chunkTransformer._destroyed ||
-            // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            (userStream as any)._writableState.ended
-          ) {
-            return next();
-          }
-          numRequestsMade = 0;
-          rowsRead++;
-          const row = this.row(rowData.key);
-          row.data = rowData.data;
-          next(null, row);
-        }),
-      ]);
+      // TODO: These lines are the changes that cause test failures.
+      // const transform = new Transform({objectMode: true});
+      // transform._transform = (rowData, _, next) => {
+      const transform = through.obj((rowData, _, next) => {
+        if (
+          chunkTransformer._destroyed ||
+          !userStream.writable
```

Comment:
Previously the line was "(userStream as any)._writableState.ended". Does this have any effect on the test failure?

Reply:
None :) I did this because on the internal streams writable seems to have the correct behavior.